src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let sum = initialValue;
```

Comment:
more of a semantic thing - `.reduce` as a method returns an `accumulator` that might be any data type - not necessarily sum (yet, yes - sum is the most often usage)
so just from the naming perspective it's `accumulator` or `result`